lib/backpex/html/layout.ex

```diff
@@ -130,6 +130,12 @@ defmodule Backpex.HTML.Layout do
       :if={Phoenix.Flash.get(@flash, :info) && Phoenix.Flash.get(@flash, :info) != ""}
       class="alert bg-info text-info-content my-4 text-sm"
       phx-value-key="info"
+      x-data="{ isVisible: false }"
+      x-init="() => { isVisible = true; setTimeout(() => isVisible = false, 5000) }"
+      x-show="isVisible"
+      x-transition:leave="transition ease-in duration-2000"
+      x-transition:leave-start="opacity-100 transform scale-100"
+      x-transition:leave-end="opacity-0 transform scale-90"
```

Comment:
The best I can do is either this: 

```
    <script>
      setTimeout(() => {
        const infoFlash = document.getElementById("info-flash");

        if (infoFlash) {
          infoFlash.style.transition = "opacity 2s ease-in";
          infoFlash.style.opacity = 0;
          setTimeout(() => infoFlash.remove(), 2000);
        }
      }, 5000);
    </script>
```
in the html in the flash_messages function(not pretty), since there is no assets with js I can't create a hook.

Or something like this: 

```
    <div
      :if={Phoenix.Flash.get(@flash, :info) && Phoenix.Flash.get(@flash, :info) != ""}
      class="alert bg-info text-info-content my-4 text-sm"
      id="info-flash"
      phx-click-away={JS.hide(to: "#info-flash", transition: {"transition ease-out duration-2000", "opacity-100 transform scale-100", "opacity-0 transform scale-90"})}
    >
```

That removes the msg when a user is using the page again. But that will remove the msg immediately on click-away with might make it disappear to fast.